### PR TITLE
Clicking on an active filter should remove it

### PR DESCRIPTION
**Pull request by andrew**
I find myself clicking on the active filter to try and toggle it, this pr fills in that behaviour so my clicks won't be in vain.

screen shot 2017-01-20 at 5 43 14 pm

### CODE REVIEW

File: app/helpers/notifications_helper.rb [hunk 1]
```diff
@@ -91,4 +91,19 @@ def filter_option(param, &block)
       end
     end
   end
+
+  def filter_link(param, value, count, &block)
```
Comment (WillAbides) [inline app/helpers/notifications_helper.rb]: 👍  moving this logic to the helper

**Comment by WillAbides** [comment]
As an aside to this, I think it would make more sense for the filter count displayed to be the number of notifications you would see if you added that filter to your current filters.

**Comment by andrew** [comment]
@WillAbides agreed, that needs some funky active record work at the controller level to get proper elasticsearch-style faceted filter counts

**Comment by andrew** [comment]
screen shot 2017-01-20 at 7 06 27 pm

Removing the filters on repos and orgs works now, although there's some slightly horrible duplication in the helpers that needs further refactoring 😬

**Pull request merged**